src/MSALWrapper.Test/Supressions.cs

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.

```

Comment:
Instead of creating a new Supression.cs file, we can use the `GlobalSupression.cs` file under the `Shared` folder. You can suppress rules for a specific project.

Reply:
Ah I avoided adding to `GlobalSuppression.cs` because it is named GlobalSuppression and implies that it applies to all projects. All our projects currently reference this so If I just add the lines to it I'm sure it would apply to all projects, which I don't want.

If there is a syntax to use here to scope it to one project that's good to know, but I still don't think we should put project specific overrides into the global file. Having a Suppressions.cs per project for any project level customization is okay IMO.

Reply:
I agree with @kyle-rader , I think we can have a different shared `GlobalSuppression.cs` for test projects.

Reply:
If it's intentional, I am okay with this. But yes, we could use scope, rather "target" even in the GlobalSupression.cs file to supress rules for a specific project.

Reply:
@kyle-rader @goagain Worth renaming to `GlobalSupressionTest.cs` file then?

Reply:
> @kyle-rader @goagain Worth renaming to `GlobalSupressionTest.cs` file then?

It's ok to me.

Reply:
Great points! I've moved the new file to the stylecop folder as `GlobalSupressions.Test.cs` - that way it can be re-used in the other test project the same way.